### DISCUSSION TITLE
編集リクエストの送信ボタンのラベルについて

### DISCUSSION THREAD

**Discussion by yimajo** [increments/qiita-discussions#162] [要望・アイデア・不具合]
編集リクエスト時の右下にある「送信」ボタンについての要望というか意見をお聞きしたいのですが、

![スクリーンショット 2022-03-04 14 23 30](https://user-images.githubusercontent.com/1137860/156704728-4dc374d8-c8c3-4cf9-bcdb-a59e6a1d2d87.png)

一度編集リクエストをしたことがある自分としては、編集リクエストにはその意図を送れるというのを知ってるので、
送信ボタンを押してしまって変更内容を書かずに「送信」してしまっていいんだろうか？
どこかに変更内容を記載するtextareaがあるんだっけ？と一瞬悩みました。

意を決して送信ボタンを押すと下記のように変更内容を書き込めるようなtextareaが出てくれて一安心...↓

![スクリーンショット 2022-03-04 14 27 19](https://user-images.githubusercontent.com/1137860/156704983-490d42fd-9bb2-4cc1-9ca5-fa7398278651.png)

ボタンラベルを「送信」と書かれてるとやり直しができない処理のように私は感じてしまったみたいです。

このような場合、「送信」ボタンじゃないほうが良いんじゃないかなと思いましたが他の方はどうでしょう？

**Comment by shiracamus** [comment]
画面キャプチャ―にある「更新する」というボタンにも違和感を感じます。これから出すのに更新？
上部に書いてある「送る」でいいのではないでしょうか。

**Comment by kskwtnk** [comment]
@yimajo @shiracamus 
ご意見ありがとうございます！
確かに、このボタンを押した瞬間に確定してしまいそう感というか、若干適切ではありませんね……
ラベル文言の見直しを検討します！ありがとうございます！
